platform/lib/templates/index.js

```diff
@@ -39,6 +40,12 @@ function createRequestContext(request={'query': {}}, context={}) {
     context.format = request.query.format;
   }
 
+  if (!ALLOWED_LEVEL.includes(request.query.level)) {
```

Comment:
nit: negated boolean expressions are harder to understand. In this case it's simple to avoid this by switching if and else.